api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -192,6 +192,15 @@ public ResponseEntity<Workspace> createWorkspace(Workspace workspace) throws Bad
     }
     final Workspace createdWorkspace = workspaceMapper.toApiWorkspace(dbWorkspace, fcWorkspace);
     workspaceAuditor.fireCreateAction(createdWorkspace, dbWorkspace.getWorkspaceId());
+
+    if (cdrVersion.getTanagraEnabled()) {
+      try {
+        workspaceService.createTanagraStudy(
+            createdWorkspace.getNamespace(), createdWorkspace.getName());
+      } catch (Exception e) {
```

Comment:
If there is an exception, how can we continue? I.E. workspace in AoU created, but studyId failed. Maybe one thing to do is when I click on the no-study-tanagra-workspace..(study creation failed), then code would check if study was created (a separate call to study table to check if the workspaceNamespace exists), if not retry to create study?

Reply:
For now we are going to log and continue. I agree that we will eventually need to handle failures better but I don't think we need to solve this right now. It will also give us time to think thru a good solution for this.